rust/tw_bitcoin/src/modules/utils.rs

```diff
@@ -0,0 +1,175 @@
+use crate::aliases::*;
+use tw_proto::BitcoinV2::Proto;
+
+// Convenience function: our protobuf library wraps certain types (such as
+// `bytes`) in `Cow`, but given that calling `clone()` on a `Cow::Borrowed(T)`
+// does not actually clone the underlying data (but the smart pointer instead),
+// we must hard-clone individual fields manually. This is unfortunately required
+// due to how protobuf library works and our use of the 'static constraints.
+pub fn hard_clone_proto_input(proto: Proto::Input<'_>) -> Proto::Input<'static> {
```

Comment:
As an option, we can disable `Cow` usage, so we'll be able to just clone the structures as there won't be lifetime parameters.
Otherwise, I also found this crate: https://docs.rs/derive-into-owned/latest/derive_into_owned/